src/control/server/mgmt_pool_test.go

```diff
@@ -56,9 +56,10 @@ func addTestPools(t *testing.T, sysdb *system.Database, poolUUIDs ...string) {
 
 	for _, uuidStr := range poolUUIDs {
 		addTestPoolService(t, sysdb, &system.PoolService{
-			PoolUUID: uuid.MustParse(uuidStr),
-			State:    system.PoolServiceStateReady,
-			Replicas: []system.Rank{0},
+			PoolUUID:  uuid.MustParse(uuidStr),
+			PoolLabel: uuidStr,
```

Comment:
IMO would be better to make this something besides the UUID. Could use the loop iteration count?